packages/sdk/src/actions/shutdownGasRelayPaymaster.test.ts

```diff
@@ -0,0 +1,65 @@
+import { expect, test } from "vitest";
+import { testActions } from "../../tests/globals.js";
+import { ALICE, WETH } from "../../tests/constants.js";
+import { prepareShutdownGasRelayPaymasterParams } from "./shutdownGasRelayPaymaster.js";
+import { toWei } from "../utils/conversion.js";
+
+test("should shutdown gas relay paymaster correctly", async () => {
+  const { comptrollerProxy } = await testActions.createTestVault({
+    vaultOwner: ALICE,
+    denominationAsset: WETH,
+  });
+
+  const depositAmount = toWei(500);
+
+  await testActions.buyShares({
+    comptrollerProxy,
+    sharesBuyer: ALICE,
+    investmentAmount: depositAmount,
+  });
+
+  const withoutGasRelayPaymaster = await testActions.getGasRelayPaymaster({
+    comptrollerProxy,
+  });
+
+  await testActions.deployGasRelayPaymaster({
+    account: ALICE,
+    address: comptrollerProxy,
+  });
+
+  const withGasRelayPaymaster = await testActions.getGasRelayPaymaster({
+    comptrollerProxy,
+  });
+
+  expect(withGasRelayPaymaster).not.toEqual(withoutGasRelayPaymaster);
+
+  await expect(
+    testActions.shutdownGasRelayPaymaster({
+      account: ALICE,
+      address: comptrollerProxy,
+    }),
```

Comment:
When testing something, you should not test it through a helper function. Helper functions should only be used in a context where the thing itself is not the test subject.